packages/components/psammead-navigation/README.md

```diff
@@ -71,6 +101,30 @@ import { latin } from '@bbc/gel-foundations/scripts';
 </Navigation>;
 ```
 
+## Dropdown Navigation Usage
+
+```jsx
+import React from 'react';
+import { CanonicalDropdown, DropdownNavigationLi } from '@bbc/psammead-navigation';
```

Comment:
Is this now in the Dropdown folder? I wonder if it's worth importign and re-exporting from the top level index.js anyway.
```suggestion
import { CanonicalDropdown, DropdownNavigationLi } from '@bbc/psammead-navigation/Dropdown';
```

Reply:
It is now in the Dropdown folder. Not sure whether it is a common practice to import and re-export components, have you seen this done before? 
Because for example in `StoryPromo`, `IndexAlsos` are in a separate subfolder but in the "Usage" section of README they are imported from the main index.

Reply:
It looks like there's a seperate package.json file for `IndexAlsos`, that means `@bbc/psammead-story-promo/index-alsos` resolves, which is where `IndexAlsos` are coming from - so that readme isn't quite right. I think we should export everything from the index personally?

Reply:
Got it, thanks! Added the imports/exports

Reply:
Sorry about the back and forth on this - a lot of other packages in psammead export in the pattern suggested by @sadickisaac and @AlistairGempf above rather than exporting everything from the index, it might be better to follow this pattern and we can update the `StoryPromo` README to avoid confusion going forward.

Reply:
@tochwill  Just to double check, all of the exports should still be in `./Dropdown` but in the README we should add an `Exports` section and also update imports in the `Usage` section?

Reply:
Yeah, that's correct. You can do the same as the index-alsos folder and put in a package.json file so it resolves as `navigation/dropdown`.

Reply:
There's a similar thing in @DenisHdz's PR: https://github.com/bbc/psammead/pull/2577/files#diff-a29c4745f8cdf126f109082527219790R1

Reply:
Thanks all, this is now updated :)